### ISSUE TITLE
Card: In high contrast black/white theme, buttons as highlighted in red are not visually recognized, they can be seen only as text.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3822]
Description: In high contrast black/white theme, buttons as highlighted in red are not visually recognized, they can be seen only as text.

Expected: UI’s shall be recognized in high contrast themes as well when accessed.

Screenshot: 

![image](https://user-images.githubusercontent.com/32538291/98773490-0e6bfe80-240f-11eb-9bb1-560a07dcd857.png)

![image](https://user-images.githubusercontent.com/32538291/98773499-1166ef00-240f-11eb-9eeb-3ff3dda4ce89.png)

![image](https://user-images.githubusercontent.com/32538291/98773508-14fa7600-240f-11eb-908e-8e888cfb7a8f.png)

**Comment by DeepakSap14** [comment]
This issue is related to styles. 
correct component is "Object marker"

**Comment by puru-hk** [comment]
Tested it is working as expected.
